docs/modules/ROOT/pages/multimap-configuration.adoc

```diff
@@ -0,0 +1,52 @@
+= Configuring MultiMaps
+:description: pass:q[Hazelcast MultiMap, also known as IMultiMap, is a specialized map where you can store multiple values under a single key. Just like any other distributed data structure implementation in Hazelcast, MultiMap is distributed and thread-safe. You can create and configure the multimaps using the `MultiMap` Custom Resource.]
+
+{description}
+
+== Configuring the `MultiMap` Resource
+
+Below are the configuration options for the `MultiMap` resource. You can find more detailed information in xref:api-ref.adoc#multimapspec[API Reference] page.
```

Comment:
```suggestion
You can use any of the following configuration options in the `MultiMap` resource. For more information, see the xref:api-ref.adoc#multimapspec[API Reference].
```
I always try to avoid saying above or below just in case the position of these options is moved and someone forgets to update the first/lead in sentence.